Angular/.stylelintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Can we merge all of our ```stylelintrc``` configs into one?

If yes, let's move this config to [the eslint config](https://github.com/DevExpress/eslint-config-devextreme) repo too, and call it ```stylelintrc.json```

If we cannot, let's move each ```stylelintrc``` separately to [the eslint config](https://github.com/DevExpress/eslint-config-devextreme), and call it ```stylelint-{framework}.json```